react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -31,6 +31,12 @@ class Thumbnail extends Component {
      */
     static propTypes = {
         _audioTrack: PropTypes.object,
+
+        /**
+         * If true, we need to render the avatar instead of the video,
+         * overriding other considerations.
+         */
+        _hideVideo: PropTypes.boolean,
```

Comment:
can you walk the extra mile here and get rid of PropTypes for flow please?

Reply:
Sure!